aula 3.html

```diff
@@ -102,83 +111,62 @@
   width:auto;
   height:auto;
   margin: 1em 10em 1em 10em;
-  
-}    
+
+}
 
 .fotopessoal{
   display:flex;
   align-items:center;
-  flex-direction: column; 
-} 
+  flex-direction: column;
+}
 
-.atalho { 
+.atalho {
   padding:0.5em;
   margin:1em;
 
 }
-
-
-nav ul {
```

Comment:
Evite definir diferentes trechos de regras css para as mesmas tags, deixa o código complicado de entender, sempre que for modificar uma tag, procure modificar o mesmo lugar. A regra de `.menu ul` ja cumpre a função de mudar esse mesmo elemento.